zerver/views/webhooks/__init__.py

```diff
@@ -0,0 +1,49 @@
+from django.conf.urls import url
+
+
+class Integration(object):
```

Comment:
Some of this stuff is webhook-specific, and some of it isn't.  

Maybe we should make a class `Integration` that has some properties (e.g. name/client/logo, with the default for logo and not for client), and then a subclass `class WebhookIntegration`  with `url`, `function`, and the default value of `client` for webhooks.